src/macros.js

```diff
@@ -659,6 +659,12 @@ defineMacro("\\mod", "\\allowbreak" +
     "\\mathchoice{\\mkern18mu}{\\mkern12mu}{\\mkern12mu}{\\mkern12mu}" +
     "{\\rm mod}\\,\\,#1");
 
+// \pmb    --   A simulation of bold.
+// It works by typesetting three copies of the argument with small offsets.
+// Ref: a rather lengthy macro in ambsy.sty
+defineMacro("\\pmb", "\\html@mathml{\\@binrel{#1}{\\mathrlap{\\mathrlap{#1}" +
+    "{\\mkern0.4mu\\raisebox0.4mu{$#1$}}}{\\mkern0.8mu#1}}}{\\mathbf{#1}}");
```

Comment:
I think it's a fluke of KaTeX that `\raisebox` works with an unbraced argument `0.4mu`.  LaTeX doesn't work that way (while TeX's `\raise` does), and we should probably fix that eventually.  So I'd be happier if we replaced with `\raisebox{0.4mu}`.

Reply:
I also feel like some of the other braces on line 666 are unnecessary, at least the first one (because of the one within `\raisebox`).  Hmm, maybe the others are necessary in case of macros in the argument.  Maybe you can also add this test:

```js
expect("\\def\\x{1}\\pmb{\\x\\def\\x{2}}").toParseLike("\\pmb{1}")
```

Reply:
Done.